app/uk/gov/hmrc/tariffclassificationfrontend/controllers/AttachmentsController.scala

```diff
@@ -44,14 +44,47 @@ class AttachmentsController @Inject()(verify: RequestActions,
                                       fileService: FileStoreService,
                                       val messagesApi: MessagesApi,
                                       implicit val appConfig: AppConfig,
-                                      implicit val mat: Materializer) extends FrontendController with I18nSupport {
+                                      implicit val mat: Materializer) extends RenderCaseAction with I18nSupport {
```

Comment:
Do you need to have the with I18nSupport here now? It's pulled in by theRenderCaseAction

Reply:
Not added that bit :)

Reply:
All of the controllers use that trait. I guess if we ever provide other languages we'd need all this